ui/src/components/credentials/IssueCredentialForm.tsx

```diff
@@ -450,16 +460,16 @@ export function IssueCredentialForm({
                         >
                           <Checkbox.Group>
                             <Space direction="vertical">
-                              <Checkbox value="SIG">
-                                <Typography.Text>Signature-based (SIG)</Typography.Text>
+                              <Checkbox value={CredentialProofType.BJJSignature2021}>
+                                <Typography.Text>Signature-based</Typography.Text>
```

Comment:
Shouldn't we show here the real proof type @demonsh?

Reply:
Yes. Please show the real proof type

Reply:
we should

Reply:
What names do you mean by proper names? `BJJSignature2021` and `Iden3SparseMerkleTreeProof` instead of `SIG` and `MTP`?

Reply:
yes, BJJSignature2021 and Iden3SparseMerkleTreeProof